include/avr/iom3000.h is missing from tarball

--- origsrc/avr-libc-1.7.0/avr/lib/avr5/Makefile.am	2010-06-16 15:03:46.000000000 -0500
+++ src/avr-libc-1.7.0/avr/lib/avr5/Makefile.am	2010-08-19 02:06:03.935931700 -0500
@@ -28,7 +28,7 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #
 
-SUBDIRS =  at90can32 at90can64 at90can128 at90pwm216 at90pwm316 at90scr100 at90usb646 at90usb647 at90usb1286 at90usb1287 at94k atmega16 atmega16a atmega161 atmega162 atmega163 atmega164a atmega164p atmega165 atmega165a atmega165p atmega168 atmega168a atmega168p atmega169 atmega169a atmega169p atmega169pa atmega16hva atmega16hva2 atmega16hvb atmega16m1 atmega16u4 atmega32 atmega323 atmega324a atmega324p atmega324pa atmega325 atmega325a atmega325p atmega3250 atmega3250a atmega3250p atmega328 atmega328p atmega329 atmega329a atmega329p atmega329pa atmega3290 atmega3290a atmega3290p atmega32c1 atmega32hvb atmega32m1 atmega32u4 atmega32u6 atmega406 atmega64 atmega640 atmega644 atmega644a atmega644p atmega644pa atmega645 atmega645a atmega645p atmega6450 atmega6450a atmega6450p atmega649 atmega649a atmega649p atmega6490 atmega6490a atmega6490p atmega64c1 atmega64hve atmega64m1 atmega128 atmega1280 atmega1281 atmega1284p atmega128rfa1 m3000
+SUBDIRS =  at90can32 at90can64 at90can128 at90pwm216 at90pwm316 at90scr100 at90usb646 at90usb647 at90usb1286 at90usb1287 at94k atmega16 atmega16a atmega161 atmega162 atmega163 atmega164a atmega164p atmega165 atmega165a atmega165p atmega168 atmega168a atmega168p atmega169 atmega169a atmega169p atmega169pa atmega16hva atmega16hva2 atmega16hvb atmega16m1 atmega16u4 atmega32 atmega323 atmega324a atmega324p atmega324pa atmega325 atmega325a atmega325p atmega3250 atmega3250a atmega3250p atmega328 atmega328p atmega329 atmega329a atmega329p atmega329pa atmega3290 atmega3290a atmega3290p atmega32c1 atmega32hvb atmega32m1 atmega32u4 atmega32u6 atmega406 atmega64 atmega640 atmega644 atmega644a atmega644p atmega644pa atmega645 atmega645a atmega645p atmega6450 atmega6450a atmega6450p atmega649 atmega649a atmega649p atmega6490 atmega6490a atmega6490p atmega64c1 atmega64hve atmega64m1 atmega128 atmega1280 atmega1281 atmega1284p atmega128rfa1
 
 AVR_TARGET          = avr5
 AVR_TARGET_DEFS     = -D__COMPILING_AVR_LIBC__
@@ -296,9 +296,6 @@ endif	# atmega1284p
 if HAS_atmega128rfa1
   AVRLIB_DEVLIST += atmega128rfa1
 endif	# atmega128rfa1
-if HAS_m3000
-  AVRLIB_DEVLIST += m3000
-endif	# m3000
 
 AM_CPPFLAGS = -I$(top_srcdir)/common -I$(top_srcdir)/include -I$(top_builddir)/include
 
